frontend/src/views/RegisterUser/RegisterUser.jsx

```diff
@@ -46,14 +48,23 @@ function RegisterUser() {
   };
 
   const registerForm = (
-    <CustomForm
-      formTitle='Sign up for an account'
-      formFields={formFields}
-      validations={validations}
-      onSubmit={onSubmit}
-      addReset={true}
-      buttonTitle='Sign Up'
-    />
+    <>
+      <CustomForm
+        formTitle='Sign Up'
+        formFields={formFields}
+        validations={validations}
+        onSubmit={onSubmit}
+        addReset={true}
```

Comment:
I think that we can have the following here and it should still work
```suggestion
        addReset
```